docs/src/modules/brandingTheme.ts

```diff
@@ -72,15 +72,15 @@ export const blueDark = {
 };
 const grey = {
   50: '#F3F6F9',
-  100: '#EAEEF3',
-  200: '#E5E8EC',
-  300: '#D7DCE1',
-  400: '#BFC7CF',
-  500: '#AAB4BE',
-  600: '#7F8E9D',
-  700: '#46505A', // contrast 8.21:1
-  800: '#2F3A45', // contrast 11.58:1
-  900: '#20262D',
+  100: '#e7ebf0',
+  200: '#e0e3e7',
+  300: '#cdd2d7',
+  400: '#b2bac2',
+  500: '#a0aab4',
+  600: '#6f7e8c',
+  700: '#3e5060', // contrast 8.21:1
+  800: '#2d3843', // contrast 11.58:1
```

Comment:
If the values have changed, are the contrast ratio comments still accurate? (or needed?)

Reply:
Updated! Also added results for APCA, the new WCAG algorithm for contrast ratios :)